.eslintrc

```diff
@@ -1,3 +1,3 @@
 {
-  "extends": "ta-webapp",
+  "extends": "ta-webapp"
```

Comment:
In JSON (which an `.eslintrc` file *looks* like), a trailing comma will break things.  I'm indifferent here since this looks like a JSON file but is apparently parsed with a YAML parser, so the trailing comma is valid.  I tried this and it is also valid:

```
{
  extends: ta-webapp
}
```
This is neither valid JSON nor JS but the YAML parser can handle it.  Let's leave trailing commas in for consistency moving forward and just leave this as is for now.